src/gui/tray/UserModel.cpp

```diff
@@ -413,15 +413,23 @@ void User::slotAddError(const QString &folderAlias, const QString &message, Erro
     }
 }
 
+bool User::isValueableActivity(Folder *folder, const SyncFileItemPtr &item)
+{
+    // Check if we are adding it to the right account and if it is useful information (protocol errors)
+    const auto correctAccount = folder->accountState() == _account.data();
+    const auto conflictFromConflictedCopy = item->_status == SyncFileItem::Conflict && Utility::isConflictFile(item->_file);
```

Comment:
@FlexW Any reason for not naming it **is**ConflictFromConflictedCopy
 ?
 If not looking into the contents of the variable, I could easily assume it's some sort of object not just boolean.

Reply:
No, in my own code I usually prefix the methods with is and remove the is from the variable names. I think in the client code we do it the other way round:)